mesh/mesh_readers.cpp

```diff
@@ -677,7 +677,7 @@ void Mesh::ReadNURBSMesh(std::istream &input, int &curved, int &read_gf)
    curved = 1;
    if (NURBSext->HavePatches())
    {
-      NURBSFECollection  *fec = new NURBSFECollection(NURBSext->GetOrder());
+      NURBSFECollection  *fec = new NURBSFECollection(1);
```

Comment:
Is `1` right here?

Reply:
This would be the only use of the "to be deleted" function above. The specification of the NURBS order in the collection is redundant as the order gets set before the actual FE gets called. This happens int the LoadFE and LoadBE routines where patch and knot vector info are also set. 

So the 1, was just a place holder. A cleaner solution would be to add a constructor that does not take any arguments. And perhaps the constructor that does take an order can be removed.

Or saver, it could be deprecated, as I can't oversee if it is essential somewhere else.